package.json

```diff
@@ -45,6 +45,7 @@
     "grunt": "^0.4.5",
     "grunt-shell": "^1.1.1",
     "gruntfile-gtx": "^0.3.0",
+    "lodash": "^3.10.1",
```

Comment:
It would appear I'd have to require _two_ modules? (`isEmpty` and `isUndefined`) I'm not sure it's possible to require both individually into `_` without doing a custom build. I'm going to remove this for now given I was only trying to make the code more readable. It would appear there are some serious drawbacks to this implementation method; I suspect it will lead to "Don't use lodash" for people working on this library (kind of a bummer).